src/globals.d.ts

```diff
@@ -13,3 +13,10 @@
 
 declare const IS_WPT_BUILD: boolean;
 declare const PACKAGE_VERSION: string;
+interface ShadowRoot {
+    adoptedStyleSheets: CSSStyleSheet[];
+}
+interface StyleSheet {
+    replace(text: string): Promise<CSSStyleSheet>;
+    replaceSync(text: string): void;
+}
```

Comment:
Are these changes necessary? My `lib.dom.d.ts` declares them already. Maybe we just need to specify a minimum typescript version or something?

Reply:
I get the following without them.

`TS2339: Property 'replace' does not exist on type 'CSSStyleSheet'.`
`TS2339: Property 'replaceSync' does not exist on type 'CSSStyleSheet'.`

For the `Stylesheet`, I believe the issue is that `lib.dom.d.ts` does indeed have them on the extended interface `CSSStyleSheet`, however, not on the base/underlying `StyleSheet` interface.

similar scenario [Property 'addRule' and 'insertRule' does not exist on type 'StyleSheet'](https://stackoverflow.com/questions/44897499/property-addrule-and-insertrule-does-not-exist-on-type-stylesheet)